sdk/core/Azure.Core.TestFramework/src/ClientTestBase.cs

```diff
@@ -143,6 +144,22 @@ protected internal virtual object InstrumentOperation(Type operationType, object
             return operation;
         }
 
+        protected internal T InstrumentOperation<T>(T operation) where T : Operation =>
+            (T)InstrumentOperation(typeof(T), operation);
```

Comment:
Why did we need to move this method up to ClientTestBase?

Reply:
Because I use it in a class derived from `ClientTestBase`, not `RecordedTestBase`.